airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/message/DestinationMessage.kt

```diff
@@ -79,13 +79,17 @@ data class Meta(
     }
 }
 
-data class DestinationRecord(
+sealed interface DestinationRecordPayload : DestinationRecordDomainMessage {
```

Comment:
Can we rename that to SerializedDestinationRecord? I feel that payload has too many other meanings. DestinationRecord might be too generic and confusing with the other types that are going to be added. Maybe TypedValueDestinationRecord

Reply:
`Payload` is the interface. I could just rename it back to `DestinationRecord`

Reply:
Renamed `DestinationRecordPayload` to `DestinationRecord`.

So now it's `DestinationRecordDomainMessage <- DestinationRecord <- DestinationRecordAirbyteValue`